rust/tests/integration/consumer.rs

```diff
@@ -679,6 +679,10 @@ fn consume_succeeds() {
                 video_pipe_consumer.app_data().downcast_ref::<()>().unwrap(),
                 &(),
             );
+            video_pipe_consumer
+                .get_stats()
+                .await
+                .expect("Failed to get consumer stats");
```

Comment:
Assuming this failed before the fix, looks good to me

Reply:
yes, it failed before.